src/app/GitUI/UserControls/FileStatusDiffCalculator.cs

```diff
@@ -83,21 +83,7 @@ private List<FileStatusWithDescription> CalculateDiffs(
                 // If the grid is filtered, parents may be rewritten
                 GitRevision actualRev = GetActualRevisionForRevision(selectedRev);
 
-                if (actualRev.ParentIds is null || actualRev.ParentIds.Count == 0)
-                {
-                    fileStatusDescs.Add(new FileStatusWithDescription(
-                        firstRev: null,
-                        secondRev: selectedRev,
-                        summary: GetDescriptionForRevision(selectedRev.ObjectId),
-                        statuses: selectedRev.TreeGuid is null
-
-                            // likely index commit without HEAD
-                            ? module.GetDiffFilesWithSubmodulesStatus(firstId: null, selectedRev.ObjectId, parentToSecond: null, cancellationToken)
-
-                            // No parent for the initial commit
-                            : module.GetTreeFiles(selectedRev.TreeGuid, full: true)));
-                }
-                else
+                if (actualRev.ParentIds?.Count is > 0)
```

Comment:
This is just a refactoring (condition equivalently inverted [and shortened]). Or have I missed something?

Reply:
correct, added when looking at it - I can revert